include/grpc/grpc_security.h

```diff
@@ -323,6 +323,11 @@ GRPCAPI grpc_channel *grpc_secure_channel_create(
     grpc_channel_credentials *creds, const char *target,
     const grpc_channel_args *args, void *reserved);
 
+GRPCAPI grpc_channel *grpc_custom_secure_channel_create(
+    void *engine, const char *target,
```

Comment:
I think they should name that `void *`. We want the compiler to prevent people from passing `char *` for example.